guides/plugins/plugins/content/cms/real-world-cms/add-real-world-cms-block.md

```diff
@@ -0,0 +1,268 @@
+# Add a real world CMS block
+
+## Overview
+
+This guide will teach you how to create a CMS block with your plugin. This guide will cover a real world example to keep everything easy we will just create a  button. The CMS block for the button we will cover right here.
+
+## Prerequisites
+
+This plugin is built based on the [Add CMS block](../add-cms-block.md) guide. We assume that you know the basics for a CMS block from this guide.
+
+## Custom block in the administration
+
+Let's get started with adding the custom block for a button in the administration. Since Shopware 6 already comes with different categories we just have to choose in which category we want to show our block. If you break down what a button is, you should come to the conclusion that a button is just text. So we create the block in the `text` category.
```

Comment:
```suggestion
Let's get started with adding the custom block for a button in the administration. Since Shopware 6 already comes with different categories, we just have to choose which category we want to show our block. If you break down what a button is, you should come to the conclusion that a button is just text. So we create the block in the `text` category.
```